### PR TITLE
Backend/rating

### PR DESCRIPTION
**Pull request by korrawe**
#140

### CODE REVIEW

File: webserver/apps/ratings/models.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from apps.lessors.models import Lessor
+
+class Rating(models.Model):
+    rating_score = models.FloatField(verbose_name='Rating Score')
+    time_stamp = models.DateTimeField(verbose_name='Time Stamp')
+    rater_name = models.CharField(verbose_name='Rater Name', max_length=100)
```
Comment (neungkl) [inline webserver/apps/ratings/models.py]: You can change the `rater_name` to foreign key of `user` (role is the same as vendor but in Django we'll use user)

File: webserver/apps/ratings/models.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    lessor = models.ForeignKey(
+        Lessor,
+        on_delete=models.CASCADE,
+        verbose_name='Lessor',
+    )
```
Comment (neungkl) [inline webserver/apps/ratings/models.py]: Is this should be `market`?

**Comment by neungkl** [comment]
More ever, this PR should have an api that you can request with `market` number. And it is return the list of rating that associated with requested market. Also, we require the api that can count how many rating in each rating score. 

```json
[
{1: 5},
{2: 10},
{3: 23},
...
]
```

This can describe as `1 star = 5 person`, `2 star = 10 person`, `3 star = 23 person`

**Requests**
- [ ] Query with market id and response with ratings that associate requested market
- [ ] Query that can count rating

File: webserver/apps/ratings/models.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from apps.lessors.models import Lessor
+
+class Rating(models.Model):
+    rating_score = models.FloatField(verbose_name='Rating Score')
```
Comment (neungkl) [inline webserver/apps/ratings/models.py]: This should be `IntegerField `

**Comment by korrawe** [comment]
The response for star counting query will be in this format:
{
    "1": 2, 
    "2": 0, 
    "3": 0, 
    "4": 0, 
    "5": 1
}

**Comment by neungkl** [comment]
Why you omitted migration files @korrawe? It's okay to push with this PR.

**Comment by korrawe** [comment]
The model view is inherited form viewsets.ModelViewSet now.

**Comment by korrawe** [comment]
Migration file added.

**Pull request merged**